apps/gateway/Taskfile.yml

```diff
@@ -4,13 +4,24 @@ vars:
   app: gateway
 
 tasks:
+  generate-schema:
+    cmds:
+      - bash ./generate-schema.sh
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Consider making the schema generation script idempotent.

Ensuring the script can be run multiple times without adverse effects increases robustness and eases automation.